src/style.css

```
@@ -1,3 +1,86 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --blue: #00ACDC;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  background: #FFF;
+  width: 100%;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: relative;
+  box-sizing: border-box;
+  left: 50px;
+  width: 40px;
+  height: 40px;
+}
```

1. box-sizing: border-box; will be moved to * selector
2. no needs to make it relative

```suggestion
.logo {
  width: 40px;
  height: 40px;
}
```